scripts/delete_bundle.py

```diff
@@ -0,0 +1,48 @@
+import sys
+import argparse
+
+import logging
+import requests
+from azul import config
+
+logger = logging.getLogger(__name__)
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(description='Delete bundles from Azul index')
+    parser.add_argument('bundles',
+                        metavar='bundle_uuid.bundle_version',
+                        type=str,
+                        nargs='+',
+                        help='bundles uuid and version to be deleted')
+    args = parser.parse_args(argv)
+    bundles = args.bundles
+    for bundle in bundles:
+        try:
+            bundle_uuid, bundle_version = bundle.split('.', 1)
+        except ValueError:
+            raise parser.parse_args(['--help'])
```

Comment:
This works? Please demo tomorrow. I'm surprised because you SHOULD only raise exception types and parse_args returns an namespace object, not an exception.